docs/user-guide/kubeconfig-file.md

```diff
@@ -83,6 +94,23 @@ users:
     client-key: path/to/my/client/key
 ```
 
+### Building your own kubeconfig file
+
+You can easily use this file as a template to create your own kubeconfig files.  First, lets explain some of the basics.
```

Comment:
We should put a notice in here that if you are deploying k8s via `kube-up.sh`, you don't need to create your own kubeconfig files, the script will do it for you.